README.md

```diff
@@ -61,19 +64,14 @@ platform-specific features:
 
 ## Building
 
-Out-of-source builds are recommended. Before anything else, do
-
-``` bash
-# --recursive is not strictly required at the moment, but a good habit to have
-git submodule update --init --recursive
-```
-
-There are some CMake options for users:
+Out-of-source builds are recommended. There are some CMake options for users:
```

Comment:
I think it is useful to be more explicit here and give an example of how to do the out of source build.  Not everyone will read the README in depth.  Some (guilty) will skip it looking for the build instructions :-)

Reply:
Do you have an idea where to put it now? The CMake discussion is split between README.md for users - where the parent project would own the build instructions, but out-of-source builds are still a good idea, and CONTRIBUTING.md for developers, where out-of-source builds seem like they should have been discussed in README.md instead.